libs/service-portal/signature-collection/src/screens/ViewList/signees.tsx

```diff
@@ -21,105 +22,142 @@ const Signees = () => {
-                    </T.Data>
-                    <T.Data></T.Data>
-                    {/*<T.Data box={{ background: boxColor }}>
-                    {person.paper && (
-                      <Icon icon="document" type="outline" color="blue400" />
-                    )}
-                    </T.Data>*/}
-                  </T.Row>
-                )
-              })}
-            </T.Body>
-          </T.Table>
-
+        signees.length > 0 ? (
           <Box marginTop={5}>
-            <Pagination
-              totalItems={listSignees.length}
-              itemsPerPage={page_size}
-              page={page}
-              renderLink={(page, className, children) => (
-                <Box
-                  cursor="pointer"
-                  className={className}
-                  onClick={() => setPage(page)}
-                  component="button"
-                >
-                  {children}
-                </Box>
-              )}
-            />
+            <T.Table>
+              <T.Head>
+                <T.Row>
+                  <T.HeadData>{formatMessage(m.signeeDate)}</T.HeadData>
+                  <T.HeadData>{formatMessage(m.signeeName)}</T.HeadData>
+                  <T.HeadData>{formatMessage(m.signeeNationalId)}</T.HeadData>
+                  <T.HeadData>{formatMessage(m.signeeAddress)}</T.HeadData>
+                  <T.HeadData></T.HeadData>
+                </T.Row>
+              </T.Head>
+              <T.Body>
+                {signees.map((s) => {
+                  const boxColor = /*person.paper ? 'purple100' : */ 'white'
+                  return (
+                    <T.Row key={s.id}>
+                      <T.Data
+                        box={{ background: boxColor }}
+                        text={{ variant: 'medium' }}
+                      >
+                        {format(new Date(), 'dd.MM.yyyy')}
+                      </T.Data>
+                      <T.Data
+                        box={{ background: boxColor }}
+                        text={{ variant: 'medium' }}
+                      >
+                        {s.signee.name}
+                      </T.Data>
+                      <T.Data
+                        box={{ background: boxColor }}
+                        text={{ variant: 'medium' }}
+                      >
+                        {formatNationalId(s.signee.nationalId)}
+                      </T.Data>
+                      <T.Data
+                        box={{ background: boxColor }}
+                        text={{ variant: 'medium' }}
+                      >
+                        {formatMessage(m.tempMessage)}
+                      </T.Data>
+                      <T.Data></T.Data>
+                      {/*<T.Data box={{ background: boxColor }}>
+                        {person.paper && (
+                          <Icon icon="document" type="outline" color="blue400" />
+                        )}
+                        </T.Data>*/}
```

Comment:
remove comment?

Reply:
im going to use it when i have the paper sign info